tests/rt.conf

```diff
@@ -410,21 +410,21 @@ RUN | rap_control_dyn64_phy32_debug                     | + hera hercules
 
 ### S2S tests ###
 COMPILE | s2swa | gnu | -DAPP=S2SWA -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 | + hera hercules | fv3 |
-RUN | cpld_control_p8                                   | + hera                               | baseline |
+RUN | cpld_control_p8                                   | + hera hercules                      | baseline |
 
 COMPILE | s2s | gnu | -DAPP=S2S -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 | + hera hercules | fv3 |
-RUN | cpld_control_nowave_noaero_p8                     | + hera                               | baseline |
+RUN | cpld_control_nowave_noaero_p8                     | + hera hercules                      | baseline |
 
 COMPILE | s2swa_debug | gnu | -DAPP=S2SWA -DDEBUG=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 | + hera hercules | fv3 |
-RUN | cpld_debug_p8                                     | + hera                               | baseline |
+RUN | cpld_debug_p8                                     | + hera hercules                      | baseline |
 
 # Unstructured WW3
 COMPILE | s2sw_pdlib | gnu | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 -DPDLIB=ON | + hera hercules | fv3 |
 RUN | cpld_control_pdlib_p8                             | + hera                               | baseline |
```

Comment:
Is ``cpld_control_pdlib_p8`` failing on hercules/GNU?

Reply:
No it passed. I skipped it for some reason.